dockerfiles/base/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Builds a base Docker image that all other pygem project images are
+# built off of (FROM). In this image we control the end user
+# experience, namely; creating the user PyGeM, setting up the users
+# environment and the default entry point and cmds.
+#
+# Authors:
```

Comment:
Please remove the list of authors. We do not follow this style and github easily shows the author of each file.